main.go

```diff
@@ -32,37 +31,22 @@ const (
 
 func main() {
 
-	ganeshaConfig := os.Getenv(ganeshaConfigEnvVar)
+	// Rad & validate config.
+	ganeshaConfig := getEnv(ganeshaConfigEnvVar, "")
 	if ganeshaConfig == "" {
 		log.Fatalf("ganesha config file must be specified with %s env var", ganeshaConfigEnvVar)
 	}
-
-	listenAddr := os.Getenv(listenAddrEnvVar)
-	if listenAddr == "" {
-		listenAddr = ":80"
-	}
-
-	dm := os.Getenv(disableMetricsEnvVar)
-	if dm == "" {
-		dm = "false"
-	}
-	disableMetrics, err := strconv.ParseBool(dm)
+	disableMetrics, err := getBoolEnv(disableMetricsEnvVar, false)
 	if err != nil {
 		log.Fatalf("%s env var value must be true or false/empty/unset", disableMetricsEnvVar)
 	}
+	listenAddr := getEnv(listenAddrEnvVar, ":80")
 
 	// All processes should start and be ready within the context timeout.  Can
-	// be extended as needed, but 10 seconds should be plenty.
-	startCtx, startCancel := context.WithTimeout(context.Background(), 10*time.Second)
+	// be extended as needed, but 30 seconds should be plenty.
+	startCtx, startCancel := context.WithTimeout(context.Background(), 30*time.Second)
 	defer startCancel()
 
-	// Start rpcbind
-	rpc := rpcbind.New()
-	rpcErrCh, err := rpc.Run()
-	if err != nil {
-		log.Fatal(err)
-	}
-
```

Comment:
Since rpcbind is no longer needed, ganesha pkg doc needs to be updated https://github.com/storageos/nfs/blob/bfd8bca8fb0ec98efc82dd1bcadac89d9dbdf7c7/ganesha/doc.go#L7-L9 .